src/pages/groupComparison/GroupComparisonMutationMapper.tsx

```diff
@@ -214,4 +382,140 @@ export default class GroupComparisonMutationMapper extends MutationMapper<
         );
         return map;
     }
+
+    @autobind
+    protected mutationsGroupedByProteinChangeForGroup(groupIndex: number) {
```

Comment:
This is purely data related, so probably shouldn't be implemented inside a component. It feels like this should be a utility function, or a computed map (keyed by group index) within a store.

Seems like we already have similar functions here like `mutationsGroupedByProteinImpactTypeForGroup` and `mutationCountsByProteinImpactTypeForGroup`. Might be better to refactor all these together in a separate PR.